### DISCUSSION TITLE
BEAMnrc and DOSXYZnrc

### DISCUSSION THREAD

**Discussion by Mona99-del** [nrc-cnrc/EGSnrc#1168] [General]
I tried using the output for BEAM_firsttrial in DOSXYZnrc by source 9, but after running the stop, as shown in the picture below, it did not show the message finished run or job failed. in BEAM_firsttrial  Some files appeared in the folder after being run in dosxyznrc, and the size of this folder was 12 GB. Please, can anyone help me?
![dosxyznrc](https://github.com/user-attachments/assets/2eb47734-091f-4c54-9d73-4ac95854e347)
![BEAM_firttrial file](https://github.com/user-attachments/assets/169ad51f-d8a4-431b-8d24-46927ac871c1)
![image](https://github.com/user-attachments/assets/8b4a7479-6854-44d0-9730-aeae48b8dca9)

**Comment by rtownson** [comment]
Set `IWATCH output` to `None`. You've selected in the inputs to store a file that tracks every interaction history, which will get very large quite quickly (the .egsgph file). You can safely delete this file. The simulation may be crashing because of the size of this file.

**Comment by Mona99-del** [comment]
Yes I deleted the file and put iwatch output  to none and when I did a new
execute on BEAMnrc and the number of history was 60000000 it took three
hours to finish and now I did a run on DOSXZYnrc but the same thing it
stopped and no message appeared even though when I used isource=1 it only
took six minutes to finish and get the 3ddose file but when I used
isource=9 it stopped at the same picture above I know on Windows does this
take a long time

**Reply by rtownson** [reply]
In DOSXYZnrc, if you surround the phantom with vacuum it can cause a crash like this. Check the value of "medsur", or the medium surrounding the phantom. You probably want to make sure it's a "1", instead of a "0", if the first material in your list of materials in the .egsphant file is air. See page 29 of the DOSXYZnrc manual for details. You need to make this change in the input file, not using the GUI, if you're using an .egsphant file (because the GUI doesn't know the materials that are in the phantom).

**Comment by Mona99-del** [comment]
thanks for the reply, yes the medium of the region outside Phantom is Vacuum I will
change to AIR700ICRu.No the first medium is water

**Reply by Mona99-del** [reply]
In DOSXYZnrc the option to edit EGSnrc parameters does not appear at the bottom. What is the reason? note that this option exists in BEAMnrc. can help me, please
![image](https://github.com/user-attachments/assets/43785729-3b33-4fca-aef5-d97ee6f7a159)

**Reply by rtownson** [reply]
It looks like the bottom of the window is cut off. There are 3 buttons at the bottom of the window, and one of them allows you to edit the EGSnrc transport parameters. The GUI windows don't resize well on small screens, unfortunately, but you could try playing with resolution or hooking up a monitor.

**Comment by Mona99-del** [comment]
Thank you very much
